### PR TITLE
logs

### PR DESCRIPTION
**Pull request by ARR552**
### What does this PR do?

Adds more info to some logs

### Reviewers

- @acroca

### CODE REVIEW

File: claimtxman/claimtxman.go [hunk 1]
```diff
@@ -235,9 +235,10 @@ func (tm *ClaimTxManager) monitorTxs(ctx context.Context) error {
 		receiptSuccessful := false
 
 		for txHash := range mTx.History {
+			mTxLog.Debug("Checking if tx %s is mined", txHash)
 			mined, receipt, err = tm.l2Node.CheckTxWasMined(ctx, txHash)
 			if err != nil {
-				mTxLog.Errorf("failed to check if tx %v was mined: %v", txHash.String(), err)
+				mTxLog.Errorf("failed to check if tx %s was mined: %v", txHash.String(), err)
```
Comment (acroca) [inline claimtxman/claimtxman.go:241]: AFAIK there's no difference between `%v` and `%s` for strings, right?

File: claimtxman/claimtxman.go [hunk 2]
```diff
@@ -418,7 +419,7 @@ func (tm *ClaimTxManager) ReviewMonitoredTx(ctx context.Context, mTx *ctmtypes.M
 		Data:  mTx.Data,
 	})
 	if err != nil {
-		err := fmt.Errorf("failed to estimate gas: %w", err)
+		err := fmt.Errorf("failed to estimate gas: %v", err)
```
Comment (acroca) [inline claimtxman/claimtxman.go:422]: Is this any different? If I'm not mistaken, the `%w` is to support `Unwrap` of the returned error, right? It shouldn't change how it's sent to the logs.

File: claimtxman/claimtxman.go (hunk 1 above)
Comment (ARR552) [reply claimtxman/claimtxman.go:241]: %v is the generic and %s is specific for string. As the variable is defined as .String() there is no doubt the type

File: claimtxman/claimtxman.go (hunk 2 above)
Comment (ARR552) [reply claimtxman/claimtxman.go:422]: Depending on how the error is created, %w can give problems. It is safer use %v which is the generic and will work in any case

**Review by acroca — APPROVED** [review]

**Pull request merged**